### DISCUSSION TITLE
How would you use @mantine/form and zod to validate a form with a Date Input that is initially empty but required

### DISCUSSION THREAD

**Discussion by escobar5** [mantinedev/mantine#4732] [Q&A]
I have a form where I have a date input, I want the date input to be required, but I don't want to set an initial value for it. As I understand, in mantine/form I should set initialValues and the ones that are not there are considered uncontrolled (https://github.com/mantinedev/mantine/issues/3512#issuecomment-1420390066)

So how do I set my form to have a required but initially empty date picker?

**Comment by rtivital** [comment]
Set the value to null

**Reply by escobar5** [reply]
If I set it to null it would give me a Typescript validation issue because the inferred type from Zod is Date (because I set the validation to be a required date)

**Reply by rtivital** [reply]
You can set values type by passing it to useForm hook – https://mantine.dev/form/values/#set-values-type
